tests/test_file.py

```diff
@@ -45,6 +46,15 @@ def test_locale(self, mozwebqa):
             print "There is no language selector on the page"
 
     @pytest.mark.nondestructive
+    def test_locales_persist_or_fallback_to_en_us(self, mozwebqa):
+        "Test loads locale and displays it or falls back to /en-US"
+        main_page = MySiteHomePage(mozwebqa)
+        locales = ['zh-TW']
+        for locale in locales:
+            urlparse.urljoin(main_page.base_url, locales)
+            Assert.true(url.endswith('locales', 'en-US'))
```

Comment:
`url` is undefined. We need to use the result of the urljoin to make an HTTP request and verify the final URL.